---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "TestResource"
    EntandoResourceNamespace: "my-namespace"
    TestResource: "my-app"
  name: "my-app-10069309"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "ba4cdb14-feed-4c69-96ec-933b339c55fd"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-08T14:57:07Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "5053600"
  uid: "ba4cdb14-feed-4c69-96ec-933b339c55fd"
lastTimestamp: "2021-06-08T14:57:07Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
